content/develop/data-types/hashes.md

```diff
@@ -77,6 +77,21 @@ encoded in special way in memory that make them very memory efficient.
 
 See the [complete list of hash commands]({{< relref "/commands/" >}}?group=hash).
 
+## Field expiration
+
+The following commands can be used to add, delete, and report on a key's individual field TTL or expiration time:
+
+* [`HEXPIRE`]({{< relref "/commands/hexpire " >}}) set the remaining time to live in seconds for a key's fields.
```

Comment:
Maybe colon after the command name or change the verb to sets, gets, etc? Probably colon is more in line with our usual imperative style.